.github/workflows/integrationTest.yml

```diff
@@ -370,42 +367,53 @@ jobs:
       - name: Verify Terraform version
         run: terraform --version
 
-      - name: Terraform init
-        run: terraform init
+      - name: Set up terraform
+        if: steps.ec2-linux-integration-test.outputs.cache-hit != 'true'
+        uses: hashicorp/setup-terraform@v1
 
+      # nick-invision/retry@v2 starts at base dir
       - name: Terraform apply
         if: steps.ec2-linux-integration-test.outputs.cache-hit != 'true'
-        run: >
-          echo run terraform and execute test code &&
-          terraform apply --auto-approve
-          -var="ssh_key=${PRIVATE_KEY}"
-          -var="github_repo=${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}.git"
-          -var="github_sha=${GITHUB_SHA}"
-          -var="install_agent=${{ matrix.arrays.installAgentCommand }}"
-          -var="user=${{ matrix.arrays.username }}"
-          -var="ami=${{ matrix.arrays.ami }}"
-          -var="ca_cert_path=${{ matrix.arrays.caCertPath }}"
-          -var="arc=${{ matrix.arrays.arc }}"
-          -var="binary_name=${{ matrix.arrays.binaryName }}"
-          -var="local_stack_host_name=${{ needs.StartLocalStack.outputs.local_stack_host_name }}"
-          -var="s3_bucket=${S3_INTEGRATION_BUCKET}"
-          -var="vpc_security_group_ids=${VPC_SECURITY_GROUPS_IDS}"
-          -var="key_name=${KEY_NAME}"
-          -var="test_name=cw-integ-test-${{ matrix.arrays.os }}"
-          -var="iam_instance_profile=${IAM_ROLE}"
-          -var="tag=${{ matrix.arrays.tag }}"
-
+        uses: nick-invision/retry@v2
+        with:
```

Comment:
Do we not want to use a retry wait time for execution?

Reply:
Not sure what you mean by this.

Reply:
Do we not want to use the `retry_wait_seconds` configuration?

Reply:
Will add